dotnet/samples/KernelSyntaxExamples/Example71_AssistantDelegation.cs

```diff
@@ -14,6 +14,10 @@
 /// </summary>
```

Comment:
Line 63 should be:

"Can you talk like a pirate?",     (notice the 'a' before pirate)

Reply:
(plus articles are exraneous)

Reply:
Then maybe put a comment to that effect?  It just looks like a typo at the moment.

Reply:
Technically its valid grammer...even if it feels slightly broken.

Reply:
Did you misspell grammar on purpose to be funny here?

Reply:
No...I'm a typo machine... =)